code/model/DNEnvironment.php

```diff
@@ -392,8 +392,10 @@ public function canDeploy($member = null) {
 		}
 		// Must be logged in to check permissions
 
-		if(Permission::checkMember($member, 'ADMIN')) {
-			return true;
+		if ($this->Usage==='Production' || $this->Usage==='Unspecified') {
```

Comment:
I'm not sure about that - this comes from a DB Enum structure, so is itself fairly constant. We don't define consts for any other state-machine enums in deployments either so it would also be inconsistent.